.makim.yaml

```diff
@@ -137,28 +203,8 @@ groups:
 
           echo "Cron jobs for LiteRev Elasticsearch have been set up successfully for user develop."
 
```

Comment:
[CRON_PATH="/opt/services/literev-elasticsearch/scripts/cronjobs"](https://github.com/thegraphnetwork-literev/es-journals/blob/3e3c7cdf325633610cdfe27013aa67a704fff046/.makim.yaml#L193)
Please, change to:
`/opt/services/es-journals/scripts/cronjobs`